jobs/v4beta1/job_search_list_tenants_test.py

```diff
@@ -13,13 +13,30 @@
 # limitations under the License.
 
 import os
+import uuid
+
+from google.cloud import talent
+import pytest
 
 import job_search_list_tenants
 
 PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
```

Comment:
Since this env var is commented out in the noxfile_config.py, Does it get assigned from somewhere else?

Reply:
It's not commenting out :)

Long story, but we're copying [`noxfile-template.py`](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile-template.py) at the project root directory to `noxfile.py`, and use it for driving the test.

noxfile.py will [override the values](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/70677d25b9bcd9b490af9150172475a089804a03/noxfile-template.py#L74) set by the build config files (e.g. [py-3.6 config](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/70677d25b9bcd9b490af9150172475a089804a03/.kokoro/python3.6/common.cfg#L50))

Reply:
I see